test/helpers.jl

```diff
@@ -2,13 +2,20 @@ const LIBC = Libdl.dlopen("libc")
 
 exists(lib::Ptr, sym::Symbol) = Libdl.dlsym_e(lib, sym) != C_NULL
 
+# https://github.com/JuliaLang/julia/pull/25646/
+@static if VERSION < v"0.7.0-DEV.3510"
```

Comment:
I'd add a comment on why this isn't... and probably won't be in Compat.

Reply:
Done